Forge/src/main/java/me/ferdz/placeableitems/event/ItemPlaceHandler.java

```diff
@@ -2,25 +2,20 @@
 
 import me.ferdz.placeableitems.block.PlaceableItemsBlock;
 import me.ferdz.placeableitems.init.PlaceableItemsMap;
-import net.minecraft.entity.Entity;
-import net.minecraft.entity.player.PlayerEntity;
 import net.minecraft.item.*;
 import net.minecraft.util.ActionResultType;
 import net.minecraft.util.math.BlockRayTraceResult;
-import net.minecraft.util.math.MathHelper;
-import net.minecraft.util.math.RayTraceContext;
-import net.minecraft.util.math.Vec3d;
-import net.minecraft.world.World;
 import net.minecraftforge.event.entity.player.PlayerInteractEvent;
-import net.minecraftforge.eventbus.api.Event;
 import net.minecraftforge.eventbus.api.SubscribeEvent;
 
 public class ItemPlaceHandler {
 
+    private boolean holdingKey = false;
+
     @SubscribeEvent
     public void onRightClickBlock(final PlayerInteractEvent.RightClickBlock e) {
-        if (!e.getPlayer().isSneaking()) { // TODO: #13 Make configurable hotkey for placing items
-            return; // Abort if the user is not sneaking
+        if (!holdingKey) { // Abort if the user is not holding the keybind
+            return;
         }
```

Comment:
One of the things I've been wondering about this issue is that I don't know if the custom keybind should place the item directly on press of the keybind, or press of the keybind + right-click. I might have to playtest this. What do you think?

Reply:
I found it most convenient to be holding the button and pressing right click. VeinMiner does something similar whereby you hold the tilde key and mine as you would normally. I think this is the least intrusive whereas having another keybind to place an item directly seems strange.

Reply:
You're right. After trying, this feels much more natural